init.lua

```diff
@@ -15,6 +15,8 @@ if mineclone_path then -- means MineClone 2 is loaded
 	moditems.boxart = "bgcolor[#d0d0d0;false]listcolors[#9d9d9d;#9d9d9d;#5c5c5c;#000000;#ffffff]"
 	moditems.trashbin_groups = {pickaxey=1,axey=1,handy=1,swordy=1,flammable=1,destroy_by_lava_flow=1,craftitem=1}
 	moditems.dumpster_groups = {pickaxey=1,axey=1,handy=1,swordy=1,flammable=0,destroy_by_lava_flow=0,craftitem=1}
+	moditems.inventory_row_size = 9
+	moditems.internal_inv_display_offset=0.5
```

Comment:
This is not necessary. Please center the list based on the formspec width and the row count.

Reply:
You're right, it's better now.